tests/io/fixtures/ritzau_feed.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0"?>
+<XMLNewsQueuedResponse xmlns="http://tempuri.org/">
+  <XMLNewsQueuedResult xmlns:i="http://www.w3.org/2001/XMLSchema-instance">
+    <error xmlns="">0</error>
+    <errormsg xmlns=""/>
+    <ListOfRBNews xmlns="">
+      <RBNews>
+        <NewsID>83e4e779-e3b7-46da-b061-34ab92bc6ec4</NewsID>
+        <PublishDate>2018-02-01T18:44:52.973</PublishDate>
```

Comment:
is this utc date?

Reply:
It is actually not specified in the documentation.